docs/content/guides/dagster/local_to_production_tutorial.mdx

```diff
@@ -0,0 +1,395 @@
+---
+title: Development to Production Workflow | Dagster
+description: This guide walks through how to transition your code from local development to production
+---
+
+# Development to Production Workflow
+
+In this example, we will walk through how to transition your assets and jobs from local development, to unit testing, and production deployments. We will cover swapping out resources and IO managers and setting run config based on the environment where your code will be executed.
```

Comment:
```suggestion
In this example, we will walk through how to transition your data pipelines from local development, to unit testing, and production deployments. We will cover swapping out resources and IO managers and setting run config based on the environment where your code will be executed.
```
I feel this guide may get referenced by someone when they're talking about general data engineering best practices (in other words, the audience of this page might be very general data practitioners who are not familiar with dagster), so "assets and jobs" may not be the most familiar term for general audience.